### DISCUSSION TITLE
Unable to utilize entire 75GB of space inspite of having it.

### DISCUSSION THREAD

**Discussion by desdemona462** [anasty17/mirror-leech-telegram-bot#1045] [Q&A] [answered]
My VPS is **50GB** but I have added block storage of **75GB**.

It's shown in the image below. But When downloading more than 50GB using the bot, it's showing **not enough storage left**.

![Screenshot 2022-09-07 005526](https://user-images.githubusercontent.com/89628603/188722115-e7fb8d27-1ddc-4244-a8a5-3f7c69a08a35.jpg)

How to fix this???

**Comment by anasty17** [comment, marked as answer]
https://stackoverflow.com/questions/24309526/how-to-change-the-docker-image-installation-directory

**Reply by desdemona462** [reply]
Thank you
